Automattic/wp-calypso
# Help Center: Move Whats New boolean to store

#### Proposed Changes

It may happen that the dot next to the Help Center icon and the one next to What's New are not aligned. This is wrong and can lead to some confusion.

This PR moves the check to see if the user has seen the What's New modal from react-query ( `useHasSeenWhatsNewModal`) to the Help Center store.

#### Testing Instructions



* Open Live Link
* Check that there is consistency between the HelpCenter icon and the More Resources link
* If you need to update the boolean value for testing, you can update your meta with `wp user meta update your-account has_seen_whats_new_modal false` in your sandbox 


Fixes #70752
